app/lib/services/simulationPasses/SimulationPassService.js

```diff
@@ -70,22 +118,36 @@ class SimulationPassService {
      */
     async getAnchorageForDataPass(dataPassId, query) {
```

Comment:
I may be missing some domain-specific info here, but I think you should not mix `anchorage data pass` and `simulation data pass`

Reply:
well, I know it is quite strange, but long time ago we agreed to keep this naming convention used somewhere else, which is `simulation passes` are anchorage(s) to `data pass` and `data passes` are anchored to `simulation pass`. Also a database design follows such convention as well as frontend one so I decided to keep it consistent here